### DISCUSSION TITLE
Select until regex match

### DISCUSSION THREAD

**Discussion by sourproton** [helix-editor/helix#7051] [Ideas] [answered]
I think it would be useful to have something similar to `f/t` but that isn't limited to a single character when in select mode.

I, for example, work a lot with REPL-based workflows and scripts that are divided into cells, like
```
# %% cell 1
print("foo")

# %% cell 2
def bar(x):
    # computes y
    y = 2 * x

    return y

print(bar(3))

# %% cell 3
print("baz")
```

It would be useful to be able to enter select mode and jump to the next `# %%`, thus selecting the whole cell.

I'm sure there are plenty of other applications to that feature.

**Comment by the-mikedavis** [comment, marked as answer]
There is an issue for this: https://github.com/helix-editor/helix/issues/5672
